internal/backends/postgresql/collection.go

```diff
@@ -241,21 +241,39 @@ func (c *collection) DeleteAll(ctx context.Context, params *backends.DeleteAllPa
 		return &backends.DeleteAllResult{Deleted: 0}, nil
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/3498
-	_ = params.RecordIDs
+	var column string
+	var placeholders []string
+	var args []any
+
+	switch params.RecordIDs {
```

Comment:
Any reason why do we use switch for nil, and default value?

Reply:
I guess to avoid `} else {`. But I'm not sure if `switch` makes it clearer in this case